src/config/settings.py

```diff
@@ -0,0 +1,56 @@
+# src/config/settings.py
+from pathlib import Path
+from pydantic import  Field
+from pydantic_settings import BaseSettings
+BASE_DIR = Path(__file__).resolve().parent.parent.parent
```

Comment:
It is unusual to resolve paths relative to `__file__`. More typically `os.path.abspath` is used.